laws/src/main/scala/cats/laws/Traverse1Laws.scala

```diff
@@ -0,0 +1,73 @@
+package cats.laws
+
+
+import cats.{Apply, Id, Semigroup, Traverse1}
+import cats.data.{Const, Nested}
+import cats.syntax.traverse1._
+import cats.syntax.reducible._
+
+trait Traverse1Laws[F[_]] extends TraverseLaws[F] with ReducibleLaws[F] {
```

Comment:
None of the laws and test were actually hit. We need to add one law test in a data type that has a vanilla `Traverse1` instance and one for a data type that uses the `NonEmptyTraverse1`.